testsuite/forge.py

```diff
@@ -689,11 +629,29 @@ def get_dashboard_link(
         if "forge" in forge_cluster_name
         else INTERN_GRAFANA_BASE_URL
     )
-    return f"{base_url}&var-namespace={forge_namespace}&var-chain_name={forge_chain_name}{grafana_time_filter}"
+    return (
+        f"{base_url}&var-namespace={forge_namespace}"
+        f"&var-chain_name={forge_chain_name}{grafana_time_filter}"
+    )
 
 
-def get_humio_logs_link(forge_namespace: str) -> str:
-    return HUMIO_LOGS_LINK.replace("$FORGE_NAMESPACE", forge_namespace)
+def get_humio_logs_link(
+    forge_namespace: str,
+    time_filter: Union[bool, Tuple[datetime, datetime]],
+) -> str:
+    filters = f"{forge_namespace}%20"
+    if time_filter is True:
+        filters += "&live=true&start=30m"
+    elif isinstance(time_filter, tuple):
```

Comment:
Does this work, I'm pretty sure a tuple will resolve to true in the previous branch?

Reply:
Disregard, Python does Python.

Reply:
is True actually checks to see if the object is equal